dynamiq/connections/connections.py

```diff
@@ -1256,10 +1256,7 @@ class MPC(BaseConnection):
         default="strict", description="The text encoding error handler."
     )
 
-    def connect(self):
-        pass
-
-    async def get_client(self):
+    async def connect(self):
```

Comment:
you don't have any async here, so just use `def connect` and after `async with self.connection.connect() as (read, write):`